package.json

```diff
@@ -17,7 +17,9 @@
     "upload-artifact": "node-riffraff-artifact",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "cdk": "cdk"
+    "cdk": "cdk",
+    "schemas": "ts-node ./src/scripts/generate-json-schema.ts",
```

Comment:
Is it worth using a verb here for clarity?

Reply:
Renamed to 'generate-schemas`